.github/workflows/build.yml

```diff
@@ -31,11 +37,11 @@ jobs:
     
     - name: Is Release?
       if: startswith(github.ref, 'refs/tags/v')
-      run: echo "##[set-env name=DEPLOY_PACKAGE;]true"
+      run: echo "::set-env name=DEPLOY_PACKAGE::true"
       
     - name: Publish to npm
       if: env.DEPLOY_PACKAGE == 'true'
-      run: npm set registry https://registry.npmjs.org/ && npm publish
+      run: npm set registry https://registry.npmjs.org/ && npm publish --access public
       env:
         NODE_AUTH_TOKEN: ${{secrets.npm_token}}      
         
```

Comment:
the next step is publishing npm pkg to github. is it working properly?

Reply:
Just verified it on my fork: 

Release: https://github.com/pruthvidhodda/js-sdk/releases
CI on master: https://github.com/pruthvidhodda/js-sdk/runs/782197467